sources/install/package_base.sh

```diff
@@ -266,33 +308,36 @@ function package_base() {
     deploy_exegol
     install_exegol-history
     fapt software-properties-common
-    add-apt-repository contrib
-    add-apt-repository non-free
+    add-repository # add-apt-repository does not work
     apt-get update
     colorecho "Starting main programs install"
-    fapt man git lsb-release pciutils pkg-config zip unzip kmod gnupg2 python2 wget \
-    gnupg2 python2-dev python3-dev python3-venv libffi-dev python3-pip zsh asciinema \
-    python-setuptools python3-setuptools npm gem automake autoconf make cmake time gcc g++ file lsof \
+    fapt man git lsb-release pciutils pkg-config zip unzip kmod gnupg2 wget \
+    python3-dev python3-venv libffi-dev python3-pip zsh asciinema \
+    python3-setuptools npm gem automake autoconf make cmake time gcc g++ file lsof \
     less x11-apps net-tools vim nano jq iputils-ping iproute2 tidy mlocate libtool \
     dos2unix ftp sshpass telnet nfs-common ncat netcat-traditional socat rdate putty \
     screen p7zip-full p7zip-rar unrar xz-utils xsltproc parallel tree ruby ruby-dev ruby-full bundler \
-    nim perl libwww-perl openjdk-17-jre openjdk-11-jre openjdk-11-jdk-headless openjdk-17-jdk-headless openjdk-11-jdk openjdk-17-jdk openvpn openresolv logrotate tmux tldr bat python3-pyftpdlib libxml2-utils \
-    virtualenv chromium libsasl2-dev python-dev libldap2-dev libssl-dev isc-dhcp-client sqlite3
+    nim perl libwww-perl openjdk-17-jre openjdk-17-jdk-headless openjdk-17-jdk openvpn openresolv logrotate tmux tldr bat python3-pyftpdlib libxml2-utils \
+    virtualenv chromium libsasl2-dev libldap2-dev libssl-dev isc-dhcp-client sqlite3 tk-dev libssl-dev
+    install_python2
+    install_python-pip                                  # Pip. Should we set pip2 to default?
+    pip install --no-cache-dir virtualenv
+
+    rm /usr/lib/python3.*/EXTERNALLY-MANAGED # https://stackoverflow.com/questions/75608323/how-do-i-solve-error-externally-managed-environment-everytime-i-use-pip3
     chsh -s /bin/zsh
 
     fapt-history dnsutils samba ssh snmp faketime
     fapt-aliases php python3 grc emacs-nox xsel
 
     install_rust_cargo
     install_rvm                                         # Ruby Version Manager
-    ln -s /bin/mkdir /usr/bin/mkdir                     # Some tools need this path for build
```

Comment:
Some tools need this path for compilation. I think this will block builds, but we'll see about that with the pipeline.